src/main/scala/nl/knaw/dans/easy/stage/Conf.scala

```diff
@@ -70,7 +70,6 @@ class Conf(args: Seq[String]) extends ScallopConf(args) {
     name = "staged-digital-object-set",
     descr = "The resulting Staged Digital Object directory (will be created if it does not exist)",
     required = true)(mayNotExist)
-  codependent(urn,doi)
```

Comment:
Not really sure why these are **not** codependent. If I understood it correctly, there is a DOI for each URN, right? Or isn't that the case?